<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Climex</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QObject{
	background-color: #555;
	color: white;
}

</string>
  </property>
  <widget class="QLabel" name="lbl_titulo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QObject{
	background-color: #999;
	color: black;
	font: 800 28pt &quot;Aria-Black&quot;;
}</string>
   </property>
   <property name="text">
    <string>  Consulta del Clima</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_buscar">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QObject{
	background-color: white;
	color: black;
	font: 87 12pt &quot;Arial&quot;;
	border-radius: 1px;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_buscar">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>70</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QObject{
	padding-bottom: 4px;
	font: 12pt &quot;Arial-black&quot;;
}
QObject:hover{
	background-color: #ccc;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../Downloads/busca.png</normaloff>../../../Downloads/busca.png</iconset>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>361</width>
     <height>271</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#ccc;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="lbl_ciudad">
      <property name="styleSheet">
       <string notr="true">color: white;
font: 800 38pt &quot;Aria-Black&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbl_grados">
      <property name="styleSheet">
       <string notr="true">color: white;
font: 800 40pt &quot;Aria-Black&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbl_caracteristica">
      <property name="styleSheet">
       <string notr="true">color: white;
font: 800 25pt &quot;Aria-Black&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
